interface/portraits/portraits_tauren_sprites.gfx

```diff
@@ -72,34 +72,125 @@ spriteTypes = {
 
 	spriteType = {
 		name = "GFX_tauren_male_base_oldage"
-		texturefile = "gfx\\characters\\tauren_male\\tauren_male_base_3.dds"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_base_1.dds"
 		noOfFrames = 1
 		norefcount = yes
 	}
 	
 	spriteType = {
 		name = "GFX_tauren_male_neck"
 		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_1.dds"
-		noOfFrames = 12
+		noOfFrames = 24
 		norefcount = yes
 		can_be_lowres = yes
 	}
 
 	spriteType = {
 		name = "GFX_tauren_male_neck_midage"
 		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_1.dds"
-		noOfFrames = 12
+		noOfFrames = 24
 		norefcount = yes
 		can_be_lowres = yes
 	}
 
 	spriteType = {
 		name = "GFX_tauren_male_neck_oldage"
-		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_3.dds"
-		noOfFrames = 12
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_1.dds"
+		noOfFrames = 24
+		norefcount = yes
+		can_be_lowres = yes
+	}
```

Comment:
You don't use these sprites. Just delete them together with other `midage` and `oldage` sprites.